xml/nsx-import-third_party.xml

```diff
@@ -259,32 +259,74 @@
-&prompt.ardana;cp vmware_nsx-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp tooz-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp python_openstackclient*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp keystoneauth1-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp python_glanceclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp python_cinderclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp oslo.utils-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp openstacksdk-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp deprecation-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp jsonpatch-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp jsonpointer-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp python_novaclient-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp pyOpenSSL-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp cryptography-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp cffi-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp pycparser-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp asn1crypto-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp ipaddress-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp warlock-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp jsonschema-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp functools32-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp httplib2-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp voluptuous-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/
-&prompt.ardana;cp tenacity-*.whl ~/third-party/vmware/venvs/vmware-neutronclient/whls/</screen>
+   <screen>&prompt.ardana;cd /tmp/stage/nsx-whls</screen>
+   <para>
+    And then copy the necessary files
+   </para>
+   <screen>&prompt.ardana;cp <replaceable>FILE_NAME</replaceable>* \
+   ~/third-party/vmware/venvs/vmware-neutronclient/whls/</screen>
+   <informaltable>
+    <tgroup cols="5">
+     <colspec colnum="1" colname="1" colwidth="20*"/>
+     <colspec colnum="2" colname="2" colwidth="20*"/>
+     <colspec colnum="3" colname="3" colwidth="20*"/>
+     <colspec colnum="4" colname="4" colwidth="20*"/>
+     <colspec colnum="5" colname="5" colwidth="20*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         <replaceable>FILE_NAMES</replaceable>
+        </para>
+        <para>
+         to copy
+        </para>
+       </entry>
+       <entry><para></para></entry>
```

Comment:
This table idea sounds good! Hadn't thought of that at all!

If you use `<entry colspan="4"/>` here, you only need one empty cell here (there also is no need to add empty paras to empty cells).

Reply:
couldn't get that to work, but it seems to look okay now.